### ISSUE TITLE
screen goes behind StatusBar

### ISSUE THREAD

**Issue by misag777** [wix/react-native-navigation#7094]
## 🐛 Bug Report

hello
 as you see the moon icon goes behind status Bar in some devices 
i tested in  samsung a10s  had issue but in  a70  and a5 works fine      
i tested in some Simulators  and it  works fine
i try hide  tab bar and  still have this issue
here is status bar default option:
`Navigation.setDefaultOptions({
  statusBar: {
    translucent: false,
    drawBehind: false,
    animated: false,
    backgroundColor: '#0075ff',
  },
});`

### Have you read the [Contributing Guidelines on issues](https://wix.github.io/react-native-navigation/docs/meta-contributing/)?
yes

## Expected behavior
 the moon icon not goes under status bar 
 
![demo 2](https://user-images.githubusercontent.com/82867079/115361027-6cc6b580-a1d5-11eb-9ece-b52b26d9ead7.jpeg)

## Actual Behavior

![demo](https://user-images.githubusercontent.com/82867079/115361202-9849a000-a1d5-11eb-9722-c9365be45b97.jpeg)

## Your Environment

- React Native Navigation version: 7.13.0
- React Native version:0.64.0
- Platform(s) (iOS, Android, or both?): android
- Device info (Simulator/Device? OS version? Debug/Release?):  android 9 samsung 10s
 both debug and release apk

**Comment by sriharshachilakapati** [comment]
We're facing the exact same issue. Any resolution for this so far?

Also, adding on to this, this is not device specific. We have tested it in emulator as well as in OnePlus 6 and OnePlus 7 Pro.

**Comment by flexsurfer** [comment]
any updates? same problem

**Comment by Vilzku** [comment]
We were able to narrow down this problem to the splash screen in a project I'm currently working on. The react-native-navigation will update the status bar after the application has loaded, causing the application content to appear beneath it after the splash screen has been removed. In our case, this problem was resolved by completely switching the splash screen library. Although there are still some issues with the splash screen moving upwards when the status bar updates, the main content is now displayed correctly with the new library.

If you use a splash screen, try removing the status bar or removing the splash completely and see if it affects the behavior.

**Comment by flexsurfer** [comment]
thanks, we use react-native-splash library, btw it affects topbar as well 
<img width="503" alt="image" src="https://user-images.githubusercontent.com/11790366/167591527-4c7f5282-63dc-488a-8149-96eb4f98a026.png">

**Comment by flexsurfer** [comment]
and it works fine in `react-native-navigation@^7.13.0:` 
but we have this issues when we upgrade to `react-native-navigation@^7.27.1:`

**Comment by flexsurfer** [comment]
yes , if splash screen is not used, then there is no issue , hm @Vilzku which splash library you're using ? thanks

**Comment by Vilzku** [comment]
We actually were also using react-native-splash-screen before and decided to replace that with **[react-native-bootsplash](https://github.com/zoontek/react-native-bootsplash)**.

**Comment by flexsurfer** [comment]
btw, in react-native-splash-screen there is second parameter (`fullscreen`) in show method, if its false then it works fine

**Comment by yedidyak** [comment]
Closing this as part of a maintainer cleanup of open issues older than one year.

Decision: Closed: stale/non-critical; no evidence it is a critical current-version RNN issue.

If this still reproduces on the latest React Native Navigation with a minimal reproduction, please open a new issue with current RN/RNN versions and repro steps.
